build_tools/update_environments_and_lock_files.py

```diff
@@ -147,23 +147,23 @@ def remove_from(alist, to_remove):
         },
     },
     {
-        "build_name": "py38_conda_defaults_openblas",
+        "build_name": "py_min_conda_defaults_openblas",
```

Comment:
I like the "min" naming scheme, but for the sake of consistency with our existing `pylatest` we should probably name it `pymin_` instead of `py_min_`.

Reply:
I knew when I wrote it :)